src/uiTest/resources/project.rhino.js

```diff
@@ -116,7 +116,10 @@ function Project(underlyingProject) {
 
```

Comment:
In .circleci/config.yml, pls modify the condition for UI tests of earlier versions so that they're run on this version as well

Reply:
In fact, I'm tempted to run them on all branches whose name include `ui-test` (as suggested to @LukaszKontowski with running them on all branches whose name contains `ide-probe`)